.circleci/config.yml

```diff
@@ -21,6 +22,12 @@ jobs:
 
     steps:
     - prometheus/setup_environment
+    - go/load-cache:
+        key: v1
+    - restore_cache:
+        keys:
+        - v1-npm-deps-{{ checksum "web/ui/react-app/yarn.lock" }}
+        - v1-npm-deps-
```

Comment:
Do we need the fallback, since `yarn.lock` should always be there?

Reply:
+1, I'm not sure exactly how this work.

Reply:
Ah ok, makes sense. :+1: